cylc/flow/command_validation.py

```diff
@@ -36,10 +36,11 @@
 from cylc.flow.task_outputs import TASK_OUTPUT_SUCCEEDED
 
 
-ERR_OPT_FLOW_VAL = (
+ERR_OPT_FLOW_VAL_ALL = (
     f"Flow values must be integers, or '{FLOW_ALL}', '{FLOW_NEW}', "
     f"or '{FLOW_NONE}'"
 )
+ERR_OPT_FLOW_VAL = f"Flow values must be integers, or '{FLOW_ALL}'"
 ERR_OPT_FLOW_VAL_2 = f"Flow values must be integers, or '{FLOW_ALL}'"
```

Comment:
The exactly duplicates the value on the next line.

Reply:
Whoops.